snuba/migrations/runner.py

```diff
@@ -418,3 +418,69 @@ def add_node(
             elif isinstance(migration, CodeMigration):
                 for python_op in migration.forwards_global():
                     python_op.execute_new_node(storage_sets)
+
+    @classmethod
+    def create_tables(
```

Comment:
Can we call this copy_tables? To ensure there is no confusion about when to use and to keep it more distinct from the existing create table operation.